### PR TITLE
update liara cli for desktop usage

### PR DESCRIPTION
**Pull request by vashian**
@mhemrg If everything is ok please make a new beta release for desktop app usage
New changes are based on the new oclif version

### CODE REVIEW

File: src/commands/account/add.ts [hunk 1]
```diff
@@ -46,15 +46,11 @@ export default class AccountAdd extends Command {
     };
 
     this.axiosConfig.baseURL = REGIONS_API_URL[region];
```
Comment (mhemrg) [inline src/commands/account/add.ts:48]: Do we still need this? Let's just remove everything related to Axios.

**Comment by mhemrg** [comment]
Published: `npm i @liara/cli@2.25.0-beta.6`

File: src/commands/account/add.ts (hunk 1 above)
Comment (vashian) [reply src/commands/account/add.ts:48]: > Do we still need this? Let's just remove everything related to Axios.

As we changed all `axios` requests to `got`, there's no reason to keep axios configuration...
I'll get rid of it.

**Comment by vashian** [comment]
shame on me 🤦🏻
I was forgotten to set time out for the `got` request....
I've cleared the project from any `axios` config.
Please review it ( more attention on `**base.ts**` )

**Comment by mhemrg** [comment]
Published: `npm i -g @liara/cli/2.25.0-beta.7`

File: src/commands/app/create.ts [hunk 2]
```diff
@@ -66,8 +56,12 @@ export default class AppCreate extends Command {
         debug(JSON.stringify(error.response.data));
       }
 
+      if (error.response && error.response.status === 404) {
+        this.error(`Could not create the app.`);
+      }
+
       if (error.response && error.response.status === 409) {
-        this.error(`The app already exists. Please use a unique name for your app.`);
+        this.error(`Another operation is already running. Please wait.`);
```
Comment (mhemrg) [inline src/commands/app/create.ts:64]: Why this has been changed? Have you rebased this branch from master?

Comment (vashian) [reply src/commands/app/create.ts:64]: I'm sure I did it...
It seems when I was resolving conflict choose the wrong change...(my mistake)
If it is possible I'll make this log error back in the new commit.

Comment (mhemrg) [reply src/commands/app/create.ts:64]: Yes please revert the error message to the previous one.

**Comment by mhemrg** [comment]
This PR should wait until #52 gets merged.

**Comment by mhemrg** [comment]
This PR has some merge conflicts.

**Comment by mhemrg** [comment]
Do we need to keep `axios` in the `package.json`?

**Comment by vashian** [comment]
> Do we need to keep `Axios` in the `package.json`?

I did not delete it completely because I want to sure my got interceptors work properly...
no, I can delete the Axios part.

**Pull request merged**